libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApiRoboTest.kt

```diff
@@ -338,47 +415,80 @@ class MapboxRouteLineApiRoboTest {
-            "[rgba, 0.0, 0.0, 0.0, 0.0], 0.0, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0], 0.09121734244852364, " +
-            "[rgba, 190.0, 160.0, 135.0, 1.0], 0.09969321187435165, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0], 0.7429050463377456, " +
-            "[rgba, 190.0, 160.0, 135.0, 1.0], 0.7533370645549629, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0], 0.7911074842354495, " +
-            "[rgba, 190.0, 160.0, 135.0, 1.0], 0.8172277935183009, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0], 0.8647541038618146, " +
-            "[rgba, 190.0, 160.0, 135.0, 1.0], 0.8661804981074899, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0], 0.8880790433163495, " +
-            "[rgba, 181.0, 130.0, 129.0, 1.0], 0.9275512210064087, " +
-            "[rgba, 134.0, 148.0, 165.0, 1.0]]"
+        val expectedPrimaryTrafficLineExpressionContents = listOf(
+            StringChecker("step"),
+            StringChecker("[line-progress]"),
+            StringChecker("[rgba, 0.0, 0.0, 0.0, 0.0]"),
+            DoubleChecker(0.0),
+            StringChecker("[rgba, 86.0, 168.0, 251.0, 1.0]"),
+            DoubleChecker(0.94254989),
+            StringChecker("[rgba, 255.0, 149.0, 0.0, 1.0]"),
+            DoubleChecker(1.0),
+            StringChecker("[rgba, 86.0, 168.0, 251.0, 1.0]")
+        )
+        val expectedAlternative1TrafficLineExpressionContents = listOf(
+            StringChecker("step"),
+            StringChecker("[line-progress]"),
+            StringChecker("[rgba, 0.0, 0.0, 0.0, 0.0]"),
+            DoubleChecker(0.0),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]"),
+            DoubleChecker(0.42786239),
+            StringChecker("[rgba, 190.0, 160.0, 135.0, 1.0]"),
+            DoubleChecker(0.4958641),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]")
+        )
+        val expectedAlternative2TrafficLineExpressionContents = listOf(
+            StringChecker("step"),
+            StringChecker("[line-progress]"),
+            StringChecker("[rgba, 0.0, 0.0, 0.0, 0.0]"),
+            DoubleChecker(0.0),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]"),
+            DoubleChecker(0.09121734),
+            StringChecker("[rgba, 190.0, 160.0, 135.0, 1.0]"),
+            DoubleChecker(0.0996932),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]"),
+            DoubleChecker(0.742905),
+            StringChecker("[rgba, 190.0, 160.0, 135.0, 1.0]"),
+            DoubleChecker(0.7533370),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]"),
+            DoubleChecker(0.7911074),
+            StringChecker("[rgba, 190.0, 160.0, 135.0, 1.0]"),
+            DoubleChecker(0.8172277),
+            StringChecker("[rgba, 134.0, 148.0, 165.0, 1.0]"),
+            DoubleChecker(0.8647541),
+            StringChecker("[rgba, 190.0, 160.0, 135.0, 1.0]"),
+            DoubleChecker(0.8661804),
```

Comment:
`0.8661804` vs `0.8661804981074899` doesn't make a big difference for the route line, right?

Reply:
Right. But I don't really get what you're implying here.